backend/server.py

```
@@ -7714,6 +7714,19 @@ async def clear_cost_attribution(user: dict = Depends(get_current_user)) -> dict
     return {"cleared": True}
 
 
+@app.get("/api/metrics/rate-limits")
+async def rate_limit_stats(user: dict = Depends(get_current_user)) -> dict[str, object]:
```

Ruff (0.15.21)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)